python/whylogs/core/preprocessing.py

```diff
@@ -265,6 +279,8 @@ def apply(data: Any) -> "PreprocessedColumn":
                     float_list.append(x)
                 elif isinstance(x, str):
                     string_list.append(x)
+                elif is_not_stub(np.ndarray) and isinstance(x, np.ndarray) and np.issubdtype(x.dtype, np.number):
```

Comment:
We probably want to try to tensorize the list itself before making a list of tensors?